README.md

```diff
@@ -19,6 +19,7 @@ Usage of mqtt-benchmark:
   -count=100: Number of messages to send per client
   -format="text": Output format: text|json
   -password="": MQTT password (empty if auth disabled)
+  -prefix="": MQTT client prefix prefix-id
```

Comment:
Let's rename it to `clientPrefix`? Otherwise can be misunderstood for payload prefix (https://github.com/krylovsk/mqtt-benchmark/pull/12/)

Reply:
What do you think about name `clientId` or `client-id`, not sure if `prefix` connotation is needed as client number is always added?

Reply:
yeah, makes sense to use hyphen and call it `client-id`.

Reply:
Fixed.